components/gridlayout/items.md

```diff
@@ -0,0 +1,384 @@
+                </TelerikDrawer>
+            </div>
+            
+        </GridLayoutItem>
+        <GridLayoutItem Column="2" Row="2">
+
+            <TelerikGrid Data="@MyData"
+                         AutoGenerateColumns="true"
+                         Height="100%"
+                         Width="100%"
+                         Pageable="true" 
+                         Sortable="true" 
+                         Groupable="true"
+                         FilterMode="Telerik.Blazor.GridFilterMode.FilterRow"
+                         Resizable="true" 
+                         Reorderable="true">
+            </TelerikGrid>
+
+        </GridLayoutItem>
+        <GridLayoutItem Column="3" Row="2">
+            <TelerikTreeView Data="@TreeViewData">
+                <TreeViewBindings>
+                    <TreeViewBinding IconClassField="@nameof(TreeViewModel.MyIconClass)"
+                                     ImageUrlField="@nameof(TreeViewModel.MyImageUrl)"
+                                     IconField="@nameof(TreeViewModel.TelerikIcon)" />
+                </TreeViewBindings>
+            </TelerikTreeView>
+        </GridLayoutItem>
+        <GridLayoutItem Column="1" ColumnSpan="3">
+            <TelerikSwitch @bind-Value="@likedLayout" Width="70px" Id="layout-liked" OnLabel="Liked" OffLabel="Not"></TelerikSwitch>
+            <label for="layout-liked">Liked!</label>
+        </GridLayoutItem>
+    </GridLayoutItems>
+</TelerikGridLayout>
+
+@code {
+    public bool likedLayout { get; set; } = true;
+
+    public List<TreeViewModel> TreeViewData { get; set; }
+
+    protected override void OnInitialized()
+    {
+        GenerateData();
+    }
+
+    public void GenerateData()
+    {
+        TreeViewData = new List<TreeViewModel>();
+
+        TreeViewData.Add(new TreeViewModel()
+        {
+            Id = 1,
+            Text = "Company",
+            ParentId = null,
+            HasChildren = true,
+            TelerikIcon = "home"
+        });
+
+        TreeViewData.Add(new TreeViewModel()
+        {
+            Id = 2,
+            Text = "Contact us",
+            ParentId = 1,
+            HasChildren = false,
+            MyIconClass = "oi oi-envelope-closed"
+        });
+
+        TreeViewData.Add(new TreeViewModel()
+        {
+            Id = 3,
+            Text = "Audio",
+            ParentId = null,
+            MyImageUrl = "https://docs.telerik.com/blazor-ui/images/speaker.png"
+        });
+    }
+
+    public class TreeViewModel
+    {
+        public int Id { get; set; }
+        public string Text { get; set; }
+        public bool HasChildren { get; set; }
+        public int? ParentId { get; set; }
+        public string TelerikIcon { get; set; }
+        public string MyIconClass { get; set; }
+        public string MyImageUrl { get; set; }
+    }
+
+    public IEnumerable<SampleData> MyData = Enumerable.Range(1, 30).Select(x => new SampleData
```

Comment:
MyData and SampleData are not intuitive names when using multiple components and multiple data sources.